purrr_basics/purrr_basics.Rmd

```diff
@@ -809,23 +1204,51 @@ class: middle, center
 
 ---
 
-## Your turn 9
+## `group_walk()`
+
+### Use `group_walk()` for side effects across groups 
+
+```{r group_walk}
+fs::dir_create(temp <- "temp")
```

Comment:
let's move `temp <- "temp"` to the line before; I prefer not to tuck away assignment actions